src/db-adapters/Mongoose/MongooseAdapter.js

```diff
@@ -202,16 +191,17 @@ export default class MongooseAdapter {
     const plural = this.inflector.plural;
 
     // Set up some data structures based on resourcesOrCollection
-    const resourceTypes = [];
-    const changeSets = {};
+    let resourceTypes = [];
+    let changeSets = {};
```

Comment:
Why are these `let` now? `const` is about object identity, so they should be perfectly fine as `const`.